### PR TITLE
OP-910 - Add print button for Operations in Main Menu \ Reports

### PR DESCRIPTION
**Pull request by nlazzer1**
See OP-910.

Add a method to the JasperReportsManager to get the operations list;
update the sql file to insert the button Operations into the menu

### CODE REVIEW

File: sql/step_73_operation_report_button.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO MENUITEM VALUES ('operationlist','angal.menu.btn.operationlist','angal.menu.operationlist','x','O','printing','org.isf.stat.gui.OperationsListLauncher','N',3);
```
Comment (dbmalkovsky) [inline sql/step_73_operation_report_button.sql:1]: There is already a step_73 (`step_73_ward_visit_duration.sql`)

This should be renamed to `step_74_operation_report_button.sql` and then added to the bottom of the file: `step_04_all_folling_steps.sql`

Comment (nlazzer1) [reply sql/step_73_operation_report_button.sql]: I fixed the commit. Thanks @dbmalkovsky

File: sql/step_04_all_following_steps.sql [hunk 2]
```diff
@@ -65,4 +65,5 @@ source step_68_update_main_menu.sql;
 source step_69_disclose_accounting_admin_features.sql;
 source step_70_increase_examination_note_field.sql;
 source step_71_drop_opd_date_vis_field.sql;
-source step_72_hospital_changes.sql;
\ No newline at end of file
+source step_72_hospital_changes.sql;
+source step_74_operation_report_button.sql;
```
[the remark this answers is not in the document]
Comment (nlazzer1) [reply sql/step_04_all_following_steps.sql]: No problem, indeed. You are like a manual of best practices :D Thanks again @dbmalkovsky

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

File: sql/step_74_operation_report_button.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO MENUITEM VALUES ('operationlist','angal.menu.btn.operationlist','angal.menu.operationlist','x','O','printing','org.isf.stat.gui.OperationsListLauncher','N',3);
+INSERT INTO GROUPMENU (GM_UG_ID_A, GM_MNI_ID_A, GM_ACTIVE) VALUES ('admin','operationlist','Y');
```
Comment (mwithi) [inline sql/step_74_operation_report_button.sql:2]: Wrong (old) value for GM_ACTIVE field

```suggestion
INSERT INTO GROUPMENU (GM_UG_ID_A, GM_MNI_ID_A, GM_ACTIVE) VALUES ('admin','operationlist','1');

```

**Pull request merged**